cypress/integration/login.js

```diff
@@ -1,27 +1,42 @@
+import LoginPage from '../objects/loginPage'
+
+
+var Namer = "Dmitrij Kovner";
+
 describe('Sourcebooks login', function() {
 
     it('Should display validation for empty user after attempted loggin', function () {
-        
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
+        const loginPage = new LoginPage()
+
+        loginPage.visit()
+
+        loginPage.getValidationError().should('not.visible')
         cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
+        loginPage.getValidationError().should('be.visible')
     })
 
-    it('Should be able to login with role User', function () {
+    //T-2
+    var Roles = ["User", "Team Lead", "Manager", "Accountant", "Admin"];
+    for(let i = 0; i < Roles.length; i++)
+    {
+        it(`Should be able to login with role ${Roles[i]}`, function () {
+            var RoleTabNumber = [1, 2, 5, 5, 6];
+            const loginPage = new LoginPage()
+            loginPage.visit();
+
+            cy.get('[id="loginForm.userId"]').click({force:true});

```

Comment:
It'd be great if you extracted this whole block into functions according to the page object model :)